### ISSUE TITLE
[jss-expand] background as array of objects

### ISSUE THREAD

**Issue by nicutimofte** [cssinjs/jss#711]
Cannot get background as array of objects. It returns `background: [object Object],[object Object];`

I have an example from the JSS Playground:

![image](https://user-images.githubusercontent.com/18091830/39889218-e9abe29a-549f-11e8-9dd6-ba0fb2945d10.png)

![image](https://user-images.githubusercontent.com/18091830/39889227-ee1e5ed4-549f-11e8-992e-c089d112c435.png)

Is it a bug or am I doing the wrong way ?

Thank you!

**Comment by nicutimofte** [comment]
I think there can be multiple background images, as you can see: 
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Using_multiple_backgrounds

**Comment by kof** [comment]
True, this syntax is simply missing in jss-expand then. Makes sense thought to add it. Feel free to submit a PR. Alternatively you can still use a string based version to implement the same.

**Comment by kof** [comment]
cc @typical000
